packages/block-library/src/group/variations.js

```diff
@@ -21,7 +22,21 @@ const variations = [
 		attributes: { layout: { type: 'flex' } },
 		scope: [ 'inserter', 'transform' ],
 		isActive: ( blockAttributes ) =>
-			blockAttributes.layout?.type === 'flex',
+			blockAttributes.layout?.type === 'flex' &&
+			( ! blockAttributes.layout?.orientation ||
+				blockAttributes.layout?.orientation === 'horizontal' ),
+		icon: row,
+	},
+	{
+		name: 'group-stack',
+		title: __( 'Stack' ),
+		description: __( 'Blocks stacked in a column.' ),
```

Comment:
I think this could be improved if it included "vertically" instead of "column" since column is already another block.

Reply:
Agreed